libnmstate/state.py

```diff
@@ -474,6 +475,13 @@ def _remove_iface_ipv6_link_local_addr(self):
                 )
             )
 
+    def _remove_ip_stack_if_disabled(self):
+        for ifstate in six.viewvalues(self.interfaces):
+            for family in (Interface.IPV4, Interface.IPV6):
+                ip_state = ifstate.get(family)
+                if ip_state and not ip_state.get(InterfaceIP.ENABLED):
```

Comment:
If `InterfaceIP.ENABLED` is not in, this will not be correct.
I think it needs to be explicit in... i.e. `ip_state[InterfaceIP.ENABLED]`